db/institutions.yml

```diff
@@ -305,14 +305,16 @@ qmul:
   state: ''
   zip: E1 4NS
 natautoumexico:
-  aka: [Universidad Nacional Autónoma de México, UNAM, unam, Autonomous National University of Mexico]
+  aka: [Universidad Nacional Autónoma de México, UNAM, unam, Autonomous National University
+        of Mexico]
   city: Mexico City
   country: Mexico
-  schools: {sciences: {name: Faculty of Sciences, aka: [Facultad de Ciencias]}}
-  departments: {physics: {name: Department of Physics, aka: [Departamiento de física, Departamiento de fisica]}}
+  departments: {physics: {name: Department of Physics, aka: [Departamiento de física,
+        Departamiento de fisica]}}
   name: National Autonomous University of Mexico
-  street: República de Brasil 33
+  schools: {sciences: {name: Faculty of Sciences, aka: [Facultad de Ciencias]}}
   state: ''
+  street: República de Brasil 33
```

Comment:
This address doesn't seem to be right for UNAM uni center, or the Faculty of Sciences or the dept of physics. I think you should use this (Coyoacán is the state):
```
Av Universidad 3000, Cd. Universitaria, Coyoacán, 04510 Ciudad de México, CDMX, Mexico
```
![image](https://user-images.githubusercontent.com/7819851/42489980-0a0eb3e4-83dc-11e8-9b71-a35d0215454c.png)

Reply:
Otherwise all the drug company entries looked fine to me.